src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
+  ToggleInspectModeAction(@NotNull FlutterView view) {
+    super(view, "Toggle Inspect Mode", "Toggle Inspect Mode", AllIcons.Actions.Menu_find);
+  }
+
+  protected void perform(AnActionEvent event) {
+    view.getFlutterApp().callBooleanExtension("ext.flutter.widgetInspector", isSelected(event));
```

Comment:
kk. Unless (or until) we can get an event when the inspector closes, we may want to treat it as a one-shot, show inspector action.